src/Pages/FeedPage/FeedPageProfile.jsx

```diff
@@ -0,0 +1,35 @@
+import useRequest from "../../hooks/useRequest";
+import { useEffect } from "react";
+function FeedPageProfile() {
```

Comment:
```suggestion
const FeedPageProfile = () => {
```
저희 컴포넌트와 함수를 화살표 함수로 이용하기로 해서 요부분 이렇게 수정해주시면 좋을것 같아요!